cli/js/lib.deno.ns.d.ts

```diff
@@ -2134,21 +2134,20 @@ declare namespace Deno {
    */
   export function metrics(): Metrics;
 
-  /** **UNSTABLE**: reconsider representation. */
-  interface ResourceMap {
-    [rid: number]: string;
-  }
-
-  /** **UNSTABLE**: The return type is under consideration and may change.
+  /** This is an internal API and as such it has `any` return type; that means
+   * it can change any time.
    *
-   * Returns a map of open _file like_ resource ids (rid) along with their string
+   * Returns a map of open resource ids (rid) along with their string
    * representations.
    *
-   *       console.log(Deno.resources()); // e.g. { 0: "stdin", 1: "stdout", 2: "stderr" }
+   *       console.log(Deno.resources());
+   *       // { 0: "stdin", 1: "stdout", 2: "stderr" }
    *       Deno.openSync('../test.file');
-   *       console.log(Deno.resources()); // e.g. { 0: "stdin", 1: "stdout", 2: "stderr", 3: "fsFile" }
+   *       console.log(Deno.resources());
+   *       // { 0: "stdin", 1: "stdout", 2: "stderr", 3: "fsFile" }
    */
-  export function resources(): ResourceMap;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  export function resources(): any;
```

Comment:
Maybe we should move this method to `Deno.internal`?

Reply:
I like it as a public api